doc/source/users_guide/testing.rst

```diff
@@ -25,6 +22,121 @@ the underlying machine and use the default compiler for that machine if this val
+
+                         This is just like an ERS test but the tasks/threading counts are modified on restart
+   PEA                  Single PE bit for bit test (default 5 days)
+                         Do an initial run on 1 PE with mpi library.     (suffix: base)
+
+                         Do the same run on 1 PE with mpiserial library. (suffix: mpiserial)
+
+                         Compare base and mpiserial.
+   PEM                  Modified PE counts for MPI(NTASKS) bit for bit test (default 5 days)
+                         Do an initial run with default PE layout                               (suffix: base)
+
+                         Do another initial run with modified PE layout (NTASKS_XXX => NTASKS_XXX/2)  (suffix: modpes)
+
+                         Compare base and modpes
+   PET                  Modified threading OPENMP bit for bit test (default 5 days)
+                         Do an initial run where all components are threaded by default. (suffix: base)
+
+                         Do another initial run with NTHRDS=1 for all components.        (suffix: single_thread)
+
+                         Compare base and single_thread.
+
+   PFS                  Performance test setup. (default 20 days)
+   ICP                  CICE performance test.
+   OCP                  POP performance test. (default 10 days)
+
+   MCC                  Multi-driver validation vs single-instance. (default 5 days)
+   NCK                  Multi-instance validation vs single instance - sequential PE for instances (default length)
+                         Do an initial run test with NINST 1. (suffix: base)
+
+                         Do an initial run test with NINST 2. (suffix: multiinst for both _0001 and _0002)
+
+                         Compare base and _0001 and _0002.
+
+   REP                  Reproducibility: Two identical runs are bit for bit. (default 5 days)
+   SBN                  Smoke build-namelist test (just run preview_namelist and check_input_data).
+   SMS                  Smoke startup test (default 5 days)
+                         Do a 5 day initial test. (suffix: base)
+   SEQ                  Different sequencing bit for bit test. (default 10 days)
+                         Do an initial run test with out-of-box PE-layout. (suffix: base)
+
+                         Do a second run where all root pes are at pe-0.   (suffix: seq)
+
+                         Compare base and seq.
+   DAE                  Data assimilation test, default 1 day, two DA cycles, no data modification.
+   PRE                  Pause-resume test: by default a bit for bit test of pause-resume cycling.
+                         Default 5 hours, five pause/resume cycles, no data modification.
+
+======================= =====================================================================================
+
+
 Each test run by create test will be put through the following mandatory phases:
```

Comment:
This should be: "Each test run by **create_test** includes the following mandatory steps:"